### ISSUE TITLE
Create a branch that removes Elements files

### ISSUE THREAD

**Issue by alawvt** [VTUL/vtechworks#536]
Create a branch that removes Elements files. See also [Create a branch without Elements scripts and deploy to vtechworks-dev](https://code.vt.edu/it-services/vtlibans_dspace/issues/98) in the vtlibans project.

**Comment by keithgee** [comment]
@soumikgh, Anne mentioned that you are working on a project to remove Elements Repository Tools 1 things that won't be needed anymore from various codebases.

My old gitlab project for the repository tools scripts hasn't been used for a while - perhaps I should delete the project since things were long moved over to our Ansible/Vagrant deployment system?

Let me know if you have any questions about anything and I'll assist to the best of my memory and notes.

Thanks,
Keith

**Comment by soumikgh** [comment]
I was just planning to remove the reference to the `elements` role from the `vtlibans_dspace` project (which is our Ansible project for setting up VTechWorks), and undo the things that the role performed.

I'm not sure if I need to do anything else. Ansible scripts are supposed to be modular that way, at least in the ideal scenario.

**Comment by alawvt** [comment]
@keithgee and @soumikgh, thank you for working on this. @keithgee 's project is https://code.vt.edu/vtkeithg/elements_install_scripts.

**Comment by alawvt** [comment]
Will need to remove from dspace/config/dspace.cfg (first added in https://github.com/VTUL/vtechworks/commit/574d07777825f150629e33ffa35f19ff12eeb6fb#diff-a60e58808bf42c545ed0dc04a4d6d5f9)
```
###############################################################################
#                        PUBLICATIONS CONFIGURATION                           #
#=============================================================================#
#                                                                             #
# This configuration is used by the Publications interface implementation     #
#                                                                             #
###############################################################################

#-----------------------------------------------------------------------------#
# Elements Configuration                                                      #
#-----------------------------------------------------------------------------#

#-----------------------------------------------------------------------------#
# URL configuration                                                           #
#-----------------------------------------------------------------------------#

# Publications Module Base URL
# ----------------------------
#
# The URL at which the publications system connector will be deployed.  This URL
# will default to:
#
# {dspace.url}/rt4ds
#
# The default URL will be incorrect if you are not deploying DSpace from the
# root of the web server context (i.e., if your dspace.url is set to
# something like http://localhost:8080/jspui, then you must set the publications
# connector url here)
#
publications.url.base = http://vtechworks.lib.vt.edu/rt4ds

# Symplectic Publications ListUpdates URL
# ---------------------------------------
#
# the URL from which the repository can request updates from the Symplectic
# Publications system
#
#publications.url.list-updates = http://192.168.1.78:9190/publications-atom/list-publications
# This is currently unused - KRG 2016-05-09
publications.url.list-updates = http://efars.provost.vt.edu:9090/publications-atom/list-publications

# Publications Module Repository URL
# ----------------------------------
#
# the repository collection url (in atom speak).  The url to which new deposits
# will be sent, and from which listings of the current holdings of the
# repository will be requested.
#
# If not specified, this url will default to:
#
# {publications.url.base}/repository
#
# publications.url.repository = http://localhost:8080/rt4ds/repository

# Publication Media Link Base URL
# -------------------------------
#
# the base media link url (in atom speak).  The url which defines the top level
# from which atom documents of publications in the repository can be obtained
#
# If not specified, this url will default to:
#
# {publications.url.base}/publication
#
# publications.url.media-link.base = http://localhost:8080/rt4ds/publication

#-----------------------------------------------------------------------------#
# Security and Authentication configuration                                   #
#-----------------------------------------------------------------------------#

# Publications Module Username
# ----------------------------
#
# the DSpace user that the publications system connector will authenticate as.
# This exists for the time being, before we implement other forms of
# authentication.  It means that the publications web service to DSpace MUST be
# IP restricted to only the machine running the publications instance that will
# talk to this service.
#
# This user will need to have rights to create items, add files and metadata to
# items, and to delete files across the whole of the DSpace instance (or the
# portion of it managed by Publications)
#
publications.username = dspace-elements-g@vt.edu
 

# IP Restricted Requests
# ----------------------
#
# The IP address from which requests will be accepted.  If this is not set, the
# application will not restrict the clients to which it responds (it will be
# assumed that the server's firewall is configured appropriately).  If it is
# set, the application will verify that incoming requests derive from the given
# IP address, and will only respond to ones which do.  This should not be
# considered a substitute for an appropriately configured firewall.
#
# publications.ip = 127.0.0.1

# Exposed Bundles
# ---------------
#
# An INCLUSIVE list of bundles which the publications connector should allow
# to be exposed to the user interface.  This allows us to prevent the display
# of publications data, full text indexes, thumbnails, etc (or to include them
# if that's what we want)
#
# A common list of options are:
#
# ORIGINAL
# LICENSE
# PUBS_DATA
# DELETED
# THUMBNAILS
#
# But the specific list will depend on your local usage of DSpace
#
publications.bundles.allowed = ORIGINAL, DELETED, LICENSE

#-----------------------------------------------------------------------------#
# Metadata configuration                                                      #
#-----------------------------------------------------------------------------#

# Publications Module Authority Controlled Metadata Fields
# --------------------------------------------------------
#
# The list of metadata fields which can be authority controlled by publications
# If specified, this list will form a core of metadata which may be subject
# to alteration by the publications metadata feed (unless the config option
# publications.metadata.update is set to false)
#
publications.metadata.authority-list = \
        dc.contributor.assignee, \
        dc.contributor.author, \
        dc.contributor.editor, \
        dc.contributor.other, \
        dc.coverage.spatial, \
        dcterms.dateAccepted, \
        dc.date.accessioned, \
        dc.date.available, \
        dc.date.issued, \
        dc.date.submitted, \
        dc.description.abstract, \
        dc.description.sponsorship, \
        dc.description.version, \
        dc.format, \
        dc.format.extent, \
        dc.format.medium, \
        dc.identifier, \
    dc.identifier.citation, \
        dc.identifier.doi, \
	dc.identifier.orcid, \
        dc.identifier.url, \
        dc.identifier.eissn, \
        dc.identifier.isbn, \
        dc.identifier.issn, \
        dc.language, \
        dc.publisher, \
        dc.relation.isPartOf, \
        dc.relation.ispartof, \
        dc.relation.replaces, \
        dc.relation.uri, \
        dc.source, \
        dc.subject, \
        dc.subject.mesh, \
        dc.title, \
        dc.type.other, \
        dc.type, \
        dcterms.references, \
    pubs.author-url, \
    pubs.awarded-date, \
        pubs.begin-page, \
        pubs.commissioning-body, \
        pubs.confidential, \
        pubs.edition, \
        pubs.end-page, \
        pubs.filed-date, \
        pubs.finish-date, \
        pubs.issue, \
        pubs.merge-from, \
        pubs.merge-to, \
        pubs.notes, \
        pubs.organisational-group, \
        pubs.patent-number, \
        pubs.patent-status, \
        pubs.repository-status, \
        pubs.place-of-publication, \
    pubs.publisher-url, \
        pubs.publication-status, \
        pubs.start-date, \
        pubs.version, \
        pubs.volume, \
        dc.description.notes, \
        dc.title.serial, \
        dc.title.volume, \
        pubs.requested-embargo-period, \
        dc.identifier.volume, \
        dc.identifier.issue, \
        dc.identifier.other, \
        dc.rights,           \
        dc.rights.uri,           \
        pubs.pii,            \
        dc.description.provenance



# Preferred Metadata Sources
# --------------------------
#
# set the list of preferred metadata records in order of preference
#
# DEPRECATED: Set in the crosswalks
#
#publications.metadata.preferred.1 = web-of-science
#publications.metadata.preferred.2 = pubmed
#publications.metadata.preferred.3 = isi-proceedings
#publications.metadata.preferred.4 = manual-entry
#publications.metadata.preferred.5 = arxiv
#publications.metadata.preferred.6 = dblp
#publications.metadata.preferred.7 = scopus

# Bibliographic data source storage
# ---------------------------------
#
# Which field to store the metadata source from which the EPrint was populated
# in
#
#publications.metadata-source.field = pubs.elements-source

# Mapping from machine-readable source names to human readable source names to
# be placed in the metadata.  If the source name is not available in the list,
# the machine readable version will be used.
#
#publications.metadata-source.mapping.web-of-science = Web of Science
#publications.metadata-source.mapping.pubmed = PubMed
#publications.metadata-source.mapping.arxiv = arXiv
#publications.metadata-source.mapping.manual-entry = Manually entered
#publications.metadata-source.mapping.dblp = DBLP
#publications.metadata-source.mapping.scopus = Scopus
#publications.metadata-source.mapping.cinii-english = CiNii-EN
#publications.metadata-source.mapping.cinii-japanese = CiNii-JP
#publications.metadata-source.mapping.wos-lite = WoS-Lite

# Metadata Field for Organisational Groups
# ----------------------------------------
#
# The metadata field which should be used to store the organisational group
# information.  If left unspecified, the organisational group information will
# not be stored.
#
# NOTE: this field must also appear in the publications.metadata.authority-list
# configuration to be correctly managed
#
publications.metadata.org-data.field = pubs.organisational-group

# Merge From Metadata Field
# -------------------------
#
# which field should items merged into a current item be registered in the
# metadata
#
publications.merge.merge-from.field = pubs.merge-from

# Merge To Metadata Field
# -----------------------
#
# which field should items merged into another item be registered in the
# metadata
#
publications.merge.merge-to.field = pubs.merge-to

# Author URL Metadata Field
# -------------------------
#
# which field should be used to store incoming author urls
#
publications.metadata.author-url.field = pubs.author-url

# Publisher URL Metadata Field
# ----------------------------
#
# which field should be used to store incoming publisher urls
#
publications.metadata.publisher-url.field = pubs.publisher-url

# Publication Type Metadata Field
# -------------------------------
#
# which field should be used to store incoming publication types
#
publications.metadata.pub-type.field = dc.type


# Publication ID Metadata Field
# -------------------------------
#
# which field should be used to store the Elements ID of a record
#
# publications.metadata.pub-id.field = pubs.elements-id

# Map from Publication Type to Metadata Field Value
# -------------------------------------------------
#
# Set up a mapping between the URLs representing the incoming publication types
# to a value to be stored in the metadata type field:
#
# publications.metadata.pub-type.field
#
# The form of the configuration is:
#
# publications.metadata.map.{type url} = {value}
#
# NOTE: the publication types are identified by URLs, which must be string
# escaped to be read into a Java Properties object.
#
publications.metadata.map.http\://www.symplectic.co.uk/publications/atom-terms/1.0/artefact = Artefact
publications.metadata.map.http\://www.symplectic.co.uk/publications/atom-terms/1.0/book = Book
publications.metadata.map.http\://www.symplectic.co.uk/publications/atom-terms/1.0/chapter = Chapter
publications.metadata.map.http\://www.symplectic.co.uk/publications/atom-terms/1.0/composition = Composition
publications.metadata.map.http\://www.symplectic.co.uk/publications/atom-terms/1.0/conference-proceeding = Conference Proceeding
publications.metadata.map.http\://www.symplectic.co.uk/publications/atom-terms/1.0/design = Design
publications.metadata.map.http\://www.symplectic.co.uk/publications/atom-terms/1.0/exhibition = Exhibition
publications.metadata.map.http\://www.symplectic.co.uk/publications/atom-terms/1.0/internet-publication = Internet Publication
publications.metadata.map.http\://www.symplectic.co.uk/publications/atom-terms/1.0/journal-article = Journal Article
publications.metadata.map.http\://www.symplectic.co.uk/publications/atom-terms/1.0/other = Other
publications.metadata.map.http\://www.symplectic.co.uk/publications/atom-terms/1.0/patent = Patent
publications.metadata.map.http\://www.symplectic.co.uk/publications/atom-terms/1.0/performance = Performance
publications.metadata.map.http\://www.symplectic.co.uk/publications/atom-terms/1.0/report = Report
publications.metadata.map.http\://www.symplectic.co.uk/publications/atom-terms/1.0/scholarly-edition = Scholarly Edition
publications.metadata.map.http\://www.symplectic.co.uk/publications/atom-terms/1.0/software = Software

# Declined publication record field
# ---------------------------------
#
# which field should be used to store information about declined publications.
# if no field is supplied, then declined information will not be stored
#
publications.metadata.declined.field = pubs.declined

# Deleted publication record field
# ---------------------------------
#
# which field should be used to store information about deleted publications.
# if no field is supplied, then deleted information will not be stored
#
publications.metadata.deleted.field = pubs.deleted

#-----------------------------------------------------------------------------#
# Crosswalk configuration                                                     #
#-----------------------------------------------------------------------------#

# DSpace PluginManager Config for XSLT Ingestion
# ----------------------------------------------
#
# Configure the ingestion crosswalk from the incoming publications metadata
# to the DSpace Internal Metadata (DIM) format.  By setting the
# XSLTIngestionCrosswalk as a self-named plugin, it will respond to the
# plugin load request if there is a configuration field
#
# NOTE: if you have crosswalks already configured you must add this to that
# configuration, instead of repeating it
#
plugin.selfnamed.org.dspace.content.crosswalk.IngestionCrosswalk = \
  org.dspace.content.crosswalk.XSLTIngestionCrosswalk

# Default XSLT Ingester
# ---------------------
#
# Configure the crosswalk stylesheets for the default crosswalk.  If a specific
# crosswalk can't be discovered for a given publication type then this default
# one will be used.
#
crosswalk.submission.SYMPLECTIC_PUBLICATIONS_NATIVE_BASIC.stylesheet = crosswalks/symplectic_xwalks_dspace_default.xsl

# Custom XSLT Ingesters
# ---------------------
#
# Configure the crosswalk stylesheets for the various publication types.
#
# NOTE: the publication types are identified by URLs, which must be string
# escaped to be read into a Java Properties object.
#
#crosswalk.submission.http\://www.symplectic.co.uk/publications/atom-terms/1.0/artefact.stylesheet = crosswalks/symplectic_xwalks_dspace_artefact.xsl
#crosswalk.submission.http\://www.symplectic.co.uk/publications/atom-terms/1.0/book.stylesheet = crosswalks/symplectic_xwalks_dspace_book.xsl
#crosswalk.submission.http\://www.symplectic.co.uk/publications/atom-terms/1.0/chapter.stylesheet = crosswalks/symplectic_xwalks_dspace_chapter.xsl
#crosswalk.submission.http\://www.symplectic.co.uk/publications/atom-terms/1.0/composition.stylesheet = crosswalks/symplectic_xwalks_dspace_composition.xsl
#crosswalk.submission.http\://www.symplectic.co.uk/publications/atom-terms/1.0/conference-proceeding.stylesheet = crosswalks/symplectic_xwalks_dspace_conference-proceeding.xsl
#crosswalk.submission.http\://www.symplectic.co.uk/publications/atom-terms/1.0/design.stylesheet = crosswalks/symplectic_xwalks_dspace_design.xsl
#crosswalk.submission.http\://www.symplectic.co.uk/publications/atom-terms/1.0/exhibition.stylesheet = crosswalks/symplectic_xwalks_dspace_exhibition.xsl
#crosswalk.submission.http\://www.symplectic.co.uk/publications/atom-terms/1.0/internet-publication.stylesheet = crosswalks/symplectic_xwalks_dspace_internet-publication.xsl
#crosswalk.submission.http\://www.symplectic.co.uk/publications/atom-terms/1.0/journal-article.stylesheet = crosswalks/symplectic_xwalks_dspace_journal-article.xsl
#crosswalk.submission.http\://www.symplectic.co.uk/publications/atom-terms/1.0/other.stylesheet = crosswalks/symplectic_xwalks_dspace_other.xsl
#crosswalk.submission.http\://www.symplectic.co.uk/publications/atom-terms/1.0/patent.stylesheet = crosswalks/symplectic_xwalks_dspace_patent.xsl
#crosswalk.submission.http\://www.symplectic.co.uk/publications/atom-terms/1.0/performance.stylesheet = crosswalks/symplectic_xwalks_dspace_performance.xsl
#crosswalk.submission.http\://www.symplectic.co.uk/publications/atom-terms/1.0/report.stylesheet = crosswalks/symplectic_xwalks_dspace_report.xsl
#crosswalk.submission.http\://www.symplectic.co.uk/publications/atom-terms/1.0/scholarly-edition.stylesheet = crosswalks/symplectic_xwalks_dspace_scholarly-edition.xsl
#crosswalk.submission.http\://www.symplectic.co.uk/publications/atom-terms/1.0/software.stylesheet = crosswalks/symplectic_xwalks_dspace_software.xsl

#-----------------------------------------------------------------------------#
# Ingest process configuration                                                #
#-----------------------------------------------------------------------------#

# Metadata Update
# ---------------
#
# after the first ingest, do we update the metadata coming from the publications
# system, or do we always leave it along
#
publications.metadata.update = true

# Licence Required to Start Workflow
# ----------------------------------
#
# should the system wait until a licence has been granted on an item before
# injecting it into the workflow?
#
publications.licence.require-for-workflow = true

# Keep Deleted Files
# ------------------
#
# What should the system do if it is asked to remove a file.  If true, the file
# will be moved to the DELETED bundle, and secured against access by anyone
# except administrators.  If false, the file will be purged from the repository
#
publications.bitstream.delete.keep-file = false

# Workflow Behaviour on File Delete
# ---------------------------------
#
# What should the system do if a delete request is received for
# a file which is already publicly available.
#
# The available options are:
#
#   delete   : delete the file anyway
#   workflow : a new item without the deleted file will be injected into the
#              workflow
#
publications.bitstream.delete.if-published = workflow

# Workflow Item Behaviour on Metadata Update
# ------------------------------------------
#
# What should the system do if a metadata update is received for an item which
# is in the workflow?  Available options are:
#
# - ignore  : updates to metadata are not applied when item is in workflow
# - nothing : the update to the metadata does not interrupt the workflow
# - reset 	: the update to the metadata causes the workflow to reset
#
publications.metadata.update.workflow = reset

# Archived Item Behaviour on Metadata Update
# ------------------------------------------
#
# What should the system do if a metadata update is received for an item which
# is in the archive?  Available options are:
#
# - ignore  : updates to metadata are not applied when item is in the archive
# - clone	: the update to the metadata causes a new item to be cloned into the
#			  workflow, ultimately a new version is archived.  If a clone
#			  already exists, this will behave as per the configuration in
#			  publications.metadata.update.workflow
# - update	: the update to the metadata causes the item in the archive to be
#			  updated with no verification
#
publications.metadata.update.archive = update

# Update List Filter
# ------------------
#
# Provide the PluginManager with an UpdatesFilter implementation which will
# determine which Publications should be queued for re-import/update
#
# The default implementation provided will discard publications which have
# been updated more recently than their last modified date, and which the
# repository has not previously heard of
#
plugin.single.uk.co.symplectic.repository.dspace.filters.UpdatesFilter = uk.co.symplectic.repository.dspace.filters.DefaultUpdatesFilter

# Collection Mapper
# -----------------
#
# Provide the PluginManager with a CollectionMapper implementation which will
# determine the collections into which to deposit an item
#
plugin.single.uk.co.symplectic.repository.dspace.collectionmapping.CollectionMapper = uk.co.symplectic.repository.dspace.collectionmapping.SpecifiedCollectionMapper

# Specified Collection Mapper Configuration
# -----------------------------------------
#
# Configure the SpecificedCollectionMapper with the handle of the primary
# collection to use.  This is the only collection into which this implementation
# of CollectionMapper will deposit
#
publications.specifiedcollectionmapper.collection = 10919/70873

# Collection Mapper update behaviour
# ----------------------------------
#
# This option controls whether collections may be reallocated when Repository Tools processes a metadata
# update from Elements.
#
# By default, this will be treated as FALSE for the SpecifiedCollectionMapper, and TRUE for the OrgDataMapper
# (This assumes that you may move the item manually if it is going into a single collection - and that collection is unlikely to change,
# whereas using the organisational mapper, you may well wish to have the mappings updated if new relationships are created in Elements)
#
# If you manually move items to other collections after they are deposited, you will want this set to false
#publications.collectionmapper.reallocateonupdate = false

#-----------------------------------------------------------------------------#
# Atom configuration                                                          #
#-----------------------------------------------------------------------------#

# Repository Organisation ID
# --------------------------
#
# the organisation id to be used in generating identifiers for atom feeds
# and entries.  It should be a string containing no spaces which identifies
# the organisation responsible for running the repository
#
publications.atom.org-id = symplectic

# Store the atom file in the repository
# -------------------------------------
# storing the atom file in the repository can be useful for debugging, or adding additional functionality locally
# for most people, it is not required, and would waste space in the repository

publications.atom.keep = false

#-----------------------------------------------------------------------------#
# Cache configuration                                                         #
#-----------------------------------------------------------------------------#

# Cache Directory
# ---------------
#
# where should publications store its cached data for operations such as
# ListHoldings
#
publications.cache.dir = ${dspace.dir}/symplectic


# symplectic elements consumer
event.consumer.elements.class = uk.co.symplectic.repository.dspace.ElementsConsumer
event.consumer.elements.filters = Item+Modify
```

**Comment by alawvt** [comment]
We need to remove files, vtechworks/dspace/config/crosswalks/
```
symplectic_xwalks_dspace_default.xsl
symplectic_xwalks_eprints_default.xsl
symplectic_xwalks_toolkit.xsl
symplectic_xwalks_toolkit_config.xsl
symplectic_xwalks_toolkit_config_eprints.xml
symplectic_xwalks_toolkit_fieldtypes.xsl
symplectic_xwalks_toolkit_fieldtypes_date.xsl
symplectic_xwalks_toolkit_formatted.xsl
symplectic_xwalks_toolkit_formatted_fieldtypes.xsl
symplectic_xwalks_toolkit_labels.xsl
symplectic_xwalks_toolkit_list.xsl
symplectic_xwalks_toolkit_list_fieldtypes.xsl
symplectic_xwalks_toolkit_mapping.xsl
symplectic_xwalks_toolkit_mapping_fieldtypes.xsl
symplectic_xwalks_toolkit_processing.xsl
symplectic_xwalks_toolkit_text.xsl
symplectic_xwalks_toolkit_utils.xsl
```
```
 vtechworks/dspace/config/export.properties
```

**Comment by keithgee** [comment]
Also, there was a database table that was created in the initial install.
Is that to be kept or dropped when updating to RT2?

**Comment by alawvt** [comment]
@keithgee, Ben Heartland from Symplectic indicated that we could remove this table. I will research it further. Thanks.

**Comment by alawvt** [comment]
Resolved with PR #545.

**Comment by alawvt** [comment]
I believe this task is complete. Thanks, @keithgee and @soumikgh.
